data/mods/Aftershock/recipes/comestible_recipes.json

```diff
@@ -2,7 +2,7 @@
   {
     "type": "recipe",
     "activity_level": "fake",
-    "result": "butter",
+    "result": "atomic_butter",
```

Comment:
The old butter recipe needs an obsoletion

Reply:
Oh yeah since it's autolearned every save already has it.  Makes sense :)